lib/orm/base_model.js

```diff
@@ -28,7 +28,37 @@ var BaseModel = function(instance_values){
             this[prop_name] = new_value;
         },
         default_getter = function(prop_name){
-            return this[prop_name];
+            var retval;
+            if(typeof fields[prop_name] != "undefined") {
+                var fieldDataType;
+                if(typeof fields[prop_name].type != "undefined") {
+                    fieldDataType = fields[prop_name].type.toLowerCase();
+                } else if(typeof fields[prop_name].toLowerCase != "undefined") {
+                    fieldDataType = fields[prop_name].toLowerCase();
+                }
+                switch(fieldDataType) {
+                    case "int":
+                    case "bigint":
+                    case "varint":
+                    case "double":
+                    case "counter":
+                        retval = parseInt(this[prop_name]);
```

Comment:
You're right, Sir. Totally forgot about JavaScript's limitations. And of course, double must have ended at the parseFloat block, if any. Will rewrite this.